config/nrfconnect/chip-module/generate_factory_data.cmake

```diff
@@ -180,7 +180,7 @@ endfunction()
 #
 function(nrfconnect_generate_factory_data)
 
-find_package(Python REQUIRED)
+find_package(Python3 REQUIRED)
```

Comment:
Does it work under windows and toolchain manager? :) Let's try..

Reply:
It should work since all other NCS Cmake scripts call it in this way, but I would appreciate it if you test it. :)